tests/Helper/ScriptsTest.php

```diff
@@ -25,12 +25,14 @@ public function test()
 
         $scripts->add('/js/last.js', 150);
         $scripts->add('/js/first.js', 50);
+        $scripts->add('/js/attr.js', 100, array('foo' => true, 'bar' => 'baz'));
```

Comment:
I have one more question. 

Do you think there is any instance that need to be something as `<script foo="true"` ? Currently the one we have is `<script foo` . Apart from that I am all good and good to merge.